README.md

```diff
@@ -603,6 +604,34 @@ targets:
   - name: pypi
 ```
 
+### Sentry internal PyPI (`sentry-pypi`)
+
+Creates a GitHub pull request to import the package into a repo set up
+like [getsentry/pypi]
+
+[getsentry/pypi]: https://github.com/getsentry/pypi
+
+**Environment**
+
+| Name           | Description                                                        |
+| -------------- | ------------------------------------------------------------------ |
+| `GITHUB_TOKEN` | Personal GitHub API token (see https://github.com/settings/tokens) |
+
+**Configuration**
+
+| Option             | Description                                                        |
+| ------------------ | ------------------------------------ |
+| `internalPypiRepo` | GitHub repo containing pypi metadata |
+
+**Example**
+
+```yaml
+targets:
+  - name: pypi
+  - name: sentry-pypi
```

Comment:
NIT: Would it make sense to change the name of this to `internal-pypi` or `alternate-pypi` instead of `sentry-pypi` if it's meant for general use outside of sentry?

Reply:
But it looks like a flow that can be generally used so I kind of agree with @mattgauntseo-sentry here.